packages/foreman/nodejs-react-dom/nodejs-react-dom.spec

```diff
@@ -1,54 +1,32 @@
 %global npm_name react-dom
 
-Name: nodejs-%{npm_name}
-Version: 16.4.0
+Name: nodejs-react-dom
+Version: 16.8.1
 Release: 1%{?dist}
 Summary: React package for working with the DOM
 License: MIT
 Group: Development/Libraries
 URL: https://reactjs.org/
-Source0: https://registry.npmjs.org/react-dom/-/react-dom-16.4.0.tgz
-Source1: https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz
-Source2: https://registry.npmjs.org/loose-envify/-/loose-envify-1.3.1.tgz
-Source3: https://registry.npmjs.org/prop-types/-/prop-types-15.6.1.tgz
-Source4: https://registry.npmjs.org/fbjs/-/fbjs-0.8.16.tgz
-Source5: https://registry.npmjs.org/js-tokens/-/js-tokens-3.0.2.tgz
-Source6: https://registry.npmjs.org/isomorphic-fetch/-/isomorphic-fetch-2.2.1.tgz
-Source7: https://registry.npmjs.org/setimmediate/-/setimmediate-1.0.5.tgz
-Source8: https://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz
-Source9: https://registry.npmjs.org/ua-parser-js/-/ua-parser-js-0.7.18.tgz
-Source10: https://registry.npmjs.org/promise/-/promise-7.3.1.tgz
-Source11: https://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.4.tgz
-Source12: https://registry.npmjs.org/node-fetch/-/node-fetch-1.7.3.tgz
-Source13: https://registry.npmjs.org/asap/-/asap-2.0.6.tgz
-Source14: https://registry.npmjs.org/encoding/-/encoding-0.1.12.tgz
-Source15: https://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.23.tgz
-Source16: https://registry.npmjs.org/is-stream/-/is-stream-1.1.0.tgz
-Source17: https://registry.npmjs.org/safer-buffer/-/safer-buffer-2.1.2.tgz
-Source18: %{npm_name}-%{version}-registry.npmjs.org.tgz
+Source0: https://registry.npmjs.org/react-dom/-/react-dom-16.8.1.tgz
+Source1: https://registry.npmjs.org/loose-envify/-/loose-envify-1.4.0.tgz
+Source2: https://registry.npmjs.org/prop-types/-/prop-types-15.7.1.tgz
+Source3: https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz
+Source4: https://registry.npmjs.org/scheduler/-/scheduler-0.13.1.tgz
+Source5: https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz
+Source6: https://registry.npmjs.org/react-is/-/react-is-16.8.1.tgz
+Source7: %{name}-%{version}-registry.npmjs.org.tgz
```

Comment:
I wonder why it changed from `%{npm_name}` to `%{name}`. The build passes, so it should be ok.

Reply:
Things have changed in a newer version of npm2rpm. This was changed because the npm_name can contain a slash which RPM can't handle. The package name is normalized and thus guaranteed to work.